core/includes/bootstrap.inc

```diff
@@ -3527,7 +3527,7 @@ function language_multilingual() {
  * @return (stdClass|string)[]
  *   An associative array of languages, keyed by the language code.
  */
-function language_list($only_enabled = FALSE, $option_list = FALSE) {
+function language_list($only_enabled = FALSE, $option_list = FALSE, $native_names = FALSE) {
```

Comment:
The new parameter needs to be added to the `@param` list in the function docblock.